Document/0x05f-Testing-Local-Authentication.md

```diff
@@ -156,7 +203,7 @@ On certain systems, it is possible to enforce the policy for biometric authentic
     keyInfo.isUserAuthenticationRequirementEnforcedBySecureHardware();
 ```
 
-##### Fingerprint Authentication using a Symmetric Key
+The following section describes how to do fingerprint authentication using a symmetric key pair.

```

Comment:
It's not a section anymore, right? I suggest this change or removing the line completely.

```suggestion
The following describes how to do fingerprint authentication using a symmetric key pair.
```